src/reason-parser/reason_attributes.ml

```diff
@@ -33,7 +33,7 @@ let rec partitionAttributes ?(partDoc=false) ?(allowUncurry=true) attrs : attrib
   | ({ attr_name = {txt="ocaml.text"}; _} as doc)::atTl when partDoc = true ->
     let partition = partitionAttributes ~partDoc ~allowUncurry atTl in
     {partition with docAttrs=doc::partition.docAttrs}
-  | ({ attr_name = {txt="ocaml.doc"}; _} as doc)::atTl when partDoc = true ->
+  | ({ attr_name = {txt="ocaml.doc" | "ocaml.text"}; _} as doc)::atTl when partDoc = true ->
```

Comment:
Hmm, another thought ... will this account for people using the shorthand form `[@text]`? I'm a bit lazy...

Reply:
No, it wouldn't. But with Reason, they probably would have no reason to ever write that explicitly because the parser has a nice `/** comment */;` syntax, right? It wouldn't be hard to cover it but it would end up converting `[@text "lkdsajf"]` into `/** comment */` syntax.

Reply:
Ah, you're right, of course this conversion is going in the other direction. 🤦 myself